src/components/ListItems/ListItemBase/index.tsx

```diff
@@ -1,14 +1,71 @@
+import { CSSProperties } from "react";
+
 import {
     ListItem as MaterialListItem,
     ListItemProps as MaterialListItemProps,
+    useTheme,
+    makeStyles,
+    Theme,
 } from "@material-ui/core";
 
 export interface ListItemBaseProps extends MaterialListItemProps {
-    button?: false | undefined;
+    variant?: "left" | "center" | "right";
+    color?: "primary" | "secondary" | "info" | "error" | "warning" | "success";
+    highlighted?: boolean;
+    hlcolor?: "primary" | "secondary" | "info" | "error" | "warning" | "success";

```

Comment:
I don't think we need this many colors, you are giving other developers too much freedom when you do it like this.
Less is more!!!

Reply:
Take some time to think, do we really need this many color for this component?